src/lib/guides/StarterKits/getting-started.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Getting Started with Wharfkit Starter Kits
+category: StarterKits
+published: true
+requiresReview: true
+---
+
+# Introduction to Wharfkit Starter Kits
+
+The Starter kits are a great way to start developing applications on Wharfkit. They offer a convenient way to install all the essential Wharfkit packages at once, while also providing `sessionKit` instances that can be used quickly instantiate sessions in your applications.
+
+The available kits are `@wharfkit/starter`, `@wharfkit/starter-eos`, `@wharfkit/starter-telos`, `@wharfkit/starter-wax`, and `@wharfkit/starter-jungle`.
+
+> **NOTE**: This guide is current as of December 2023 and reflects the latest versions of Wharfkit Starter Kits.
```

Comment:
I don't think this is needed, this was included in other guides when we were pre-release.